documentation/largescaledeployment.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Recommended configuration for large scale cluster deployment
 ---
-vSphere Cloud Provider is successfully validated with 500 nodes Kubernetes cluster.
+vSphere Cloud Provider is successfully validated with 1000 nodes Kubernetes cluster.
```

Comment:
Let's make this info easy to find by making this **bold**.
Maximum scale validated - 1000 nodes Kubernetes cluster.